includes/shareable-content.php

```diff
@@ -132,28 +106,28 @@
 					)
 				)
 			);
-		echo '</div>'; // .cc-license
-		echo wp_kses_post( $license_statement );
-	echo '</div>'; // .cc-policy
-
-	// what we display to the embedder
-	echo '<div class="article-info">';
-		echo wp_kses_post( $article_info );
-	echo '</div>'; // .article-info
-
-	// the text area that is copyable
-	echo wp_kses_post(
-		sprintf(
-			'<textarea readonly id="republication-tracker-tool-shareable-content" rows="5">%1$s %2$s %3$s</textarea>',
-			esc_html( $article_info ),
-			$content . "\n\n",
-			wpautop( $attribution_statement . htmlentities( $pixel ) )
-		)
-	);
-	if ( ! $is_amp ) {
-		?>
+			echo '</div>'; // .cc-license
+			echo wp_kses_post( $license_statement );
+			echo '</div>'; // .cc-policy
+
+			// what we display to the embedder
+			echo '<div class="article-info">';
+			echo wp_kses_post( $article_info );
+			echo '</div>'; // .article-info
+
+			// the text area that is copyable
+			echo wp_kses_post(
+				sprintf(
+					'<textarea readonly id="republication-tracker-tool-shareable-content" rows="5">%1$s %2$s %3$s</textarea>',
+					esc_html( $article_info ),
+					$content . "\n\n",
+					wpautop( $attribution_statement . htmlentities( $pixel ) )
```

Comment:
The `wpautop` will result in the `$pixel` being wrapped in `<p>` tags, meaning it creates some extra whitespace if the `p` has margins and/or padding. Not a huge deal, but worth pointing out.

Reply:
Good catch - fixed to avoid wrapping in a paragraph if there's no attribution statement in 7f252805312290636edea74b2cda56718bfe97eb